backend/core/management/commands/populate-periodic-tasks.py

```diff
@@ -0,0 +1,65 @@
+from django.core.management.base import BaseCommand
+from django_celery_beat.models import CrontabSchedule, PeriodicTask, IntervalSchedule
+
+
+class Command(BaseCommand):
```

Comment:
Could you try and add a test for this command? e.g. 

- Launch the command
- Check that X `CrontabSchedule`s and Y `PeriodicTask`s have been created

Thanks!